demisto_sdk/commands/common/docker_util.py

```diff
@@ -0,0 +1,180 @@
+import io
+import os
+import tarfile
+from typing import Dict, List, Optional, Union
+
+import docker
+from demisto_sdk.commands.common.logger import logger
+
+client = docker.from_env()
+
+
+class ContainerRunner:
+    def __init__(self, image: str, container_name: Optional[str] = None):
+        """
+
+        Args:
+            image: the image (name or id)
+            container_name: you can give your container a meaningful name (optional)
+        """
+        self._base_image_name = image
+        self._image_name = image
+        self.container_name = container_name
+        self._container_obj: Optional[docker.models.containers.Container] = None
+
+    @property
+    def container(self) -> docker.models.containers.Container:
+        """
+        Notes:
+            try to avoid using the container in this way.
```

Comment:
the reason for that is because it’s possible to break the logic (e.g. it’s possible to run something without commit after and so).
but I didn’t want to hide the options that the docker SDK give us.